### DISCUSSION TITLE
Zilzal Classic

### DISCUSSION THREAD

**Discussion by MesopotamianGroyper** [iptv-org/community#679] [Q&A]
**Channel Name:** Zilzal Classic
**Channel ID:** ZilzalClassic.eg
**Country:** Egypt
**Language:** Arabic
**Website:** https://www.lyngsat.com/tvchannels/--/Zilzal-Classic.html
**Stream URL (optional):** xxx
**Notes (optional):** While the channel exists, I am not sure about its official website. If it is possible to add it, that would be great. If not, then fair enough. Thank you.

**Comment by Dum4G** [comment]
Can't find any source for the playlist. Meanwhile you can watch it here
https://tv.taghtia.com/en/watch/zilzal-classic

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19677
